extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -539,64 +540,64 @@ private function processAllNotInTrack() {
+			if( $this->mTitle->getNamespace() == NS_USER && $this->mTitle->getText() == $this->mUser->getName() ) {
 				$this->awardNotInTrackBadge( BADGE_INTRODUCTION );
 			}
 
 		}
 		// BADGE_SAYHI
-		if ( !$this->hasBadge( BADGE_SAYHI ) ) {
-			if ( ( $this->mTitle->getNamespace() == NS_USER_TALK || (  defined( "NS_USER_WALL_MESSAGE" ) && $this->mTitle->getNamespace() == NS_USER_WALL_MESSAGE  ) )&& $this->mTitle->getBaseText() != $this->mUser->getName() ) {
+		if( !$this->hasBadge( BADGE_SAYHI ) ) {
+			if( ( $this->mTitle->getNamespace() == NS_USER_TALK || ( defined( "NS_USER_WALL_MESSAGE" ) && $this->mTitle->getNamespace() == NS_USER_WALL_MESSAGE ) ) && $this->mTitle->getBaseText() != $this->mUser->getName() ) {
 				$this->awardNotInTrackBadge( BADGE_SAYHI );
 			}
 
 		}
 
 		// BADGE_POUNCE
-		if ( !$this->hasBadge( BADGE_POUNCE ) ) {
-			if ( $this->mTitle->isContentPage() && $this->mStatus->value['new'] != true ) {
-				if ( empty( $this->mCounters[BADGE_POUNCE] ) || !in_array( $this->mArticle->getID(), $this->mCounters[BADGE_POUNCE] ) ) {
+		if( !$this->hasBadge( BADGE_POUNCE ) ) {
+			if( $this->mTitle->isContentPage() && $this->mStatus->value[ 'new' ] != true ) {
+				if( empty( $this->mCounters[ BADGE_POUNCE ] ) || !in_array( $this->mArticle->getID(), $this->mCounters[ BADGE_POUNCE ] ) ) {
 					$firstRevision = $this->mTitle->getFirstRevision();
 
-					if ( $firstRevision instanceof Revision && ( strtotime( wfTimestampNow() ) - strtotime( $firstRevision->getTimestamp() ) < ( 3600 /* 1h */ ) ) ) {
-						if ( empty( $this->mCounters[BADGE_POUNCE] ) ) {
-							$this->mCounters[BADGE_POUNCE] = array();
+					if( $firstRevision instanceof Revision && ( strtotime( wfTimestampNow() ) - strtotime( $firstRevision->getTimestamp() ) < ( 3600 /* 1h */ ) ) ) {
+						if( empty( $this->mCounters[ BADGE_POUNCE ] ) ) {
+							$this->mCounters[ BADGE_POUNCE ] = array();
 						}
-						$this->mCounters[BADGE_POUNCE][] = $this->mArticle->getID();
-						if ( count( $this->mCounters[BADGE_POUNCE] ) > 99 && !$this->hasBadge( BADGE_POUNCE ) ) {
+						$this->mCounters[ BADGE_POUNCE ][] = $this->mArticle->getID();
+						if( count( $this->mCounters[ BADGE_POUNCE ] ) > 99 && !$this->hasBadge( BADGE_POUNCE ) ) {
 							// badge is awarded when user makes their 100th edit to a unique article within 1h of it's creation
 							// 99 was put here to correct an error in awarding that did not affect counters and award missing badges
 							$this->awardNotInTrackBadge( BADGE_POUNCE );
-							unset( $this->mCounters[BADGE_POUNCE] );
+							unset( $this->mCounters[ BADGE_POUNCE ] );
 						}
 					}
 				}
 			}
 		}
 
 		// BADGE_CAFFEINATED
-		if ( !$this->hasBadge( BADGE_CAFFEINATED ) ) {
-			if ( $this->mTitle->isContentPage() ) {
-				if ( empty( $this->mCounters[BADGE_CAFFEINATED] ) ) {
-					$this->mCounters[BADGE_CAFFEINATED] = array( COUNTERS_COUNTER => 1 );
+		if( !$this->hasBadge( BADGE_CAFFEINATED ) ) {
+			if( $this->mTitle->isContentPage() ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)